main.js

```diff
@@ -0,0 +1,24 @@
+var menu = document.querySelector(".menu-mobile");
+var ham = document.querySelector(".ham");
+var xIcon = document.querySelector(".xIcon");
+var menuIcon = document.querySelector(".menuIcon");
+
+ham.addEventListener("click", toggleMenu);
+
+function toggleMenu() {
```

Comment:
This function was used before it was declared. It'll work since function declarations are hoisted (https://www.youtube.com/watch?v=j-9_15QBW2s&ab_channel=ColtSteele) but in this case I'd declare the function before using it. :)